src/networkrequest.cpp

```diff
@@ -959,7 +964,13 @@ void NetworkRequest::handleReply(QNetworkReply* reply) {
   connect(m_reply, &QNetworkReply::finished, this,
           &NetworkRequest::maybeDeleteLater);
   connect(m_reply, &QNetworkReply::downloadProgress, this,
-          &NetworkRequest::requestUpdated);
+          [&](qint64 bytesReceived, qint64 bytesTotal) {
+            if (m_discardData) {
+              m_reply->readAll();
```

Comment:
If I’m understanding correctly this would prevent us from storing downloaded data in memory. Is this a feasible approach?